src/components/application_manager/include/application_manager/resumption/resumption_data_processor.h

```diff
@@ -125,6 +125,17 @@ class ResumptionDataProcessor : public app_mngr::event_engine::EventObserver {
                        const hmi_apis::FunctionID::eType);
 
  private:
+  utils::Optional<uint32_t> GetAppIdByRequestId(
```

Comment:
GetAppIdWaitingForRequest()

Reply:
Done in 90d5b8f7a4d8ee35f2581eff4cb63a2126a04f08